### DISCUSSION TITLE
Release-5.2.0 Neo4j startup issue

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#549] [Bugs] [answered]
Hi team!
@santhosh-tg 
In neo4j provisioning, there is no startup commands, and there is a startup job inside a deployment( Deploy/dev/KnowledgePlatform/StartNeo4jCluster).  But in docs,  this job exection details are not available.

Regards,
Gohila

**Comment by chitranshu-keshav** [comment]
@gandham-santhosh Can you please get the document updated and close this thread

**Reply by santhosh-tg** [reply]
@gandham-santhosh @pallakartheekreddy  Please review https://github.com/Sunbird-Knowlg/sunbird-learning-platform/pull/1957

**Reply by gandham-santhosh** [reply]
@pallakartheekreddy I reviewed the PR. Can you please check and merge the same?

**Reply by pallakartheekreddy** [reply]
@santhosh-tg @gandham-santhosh Can you please raise this PR to the latest branch.

**Reply by Krishnaj20** [reply]
@santhosh-tg - Is this resolved? If so pls mark it as answered.

**Reply by santhosh-tg** [reply]
Hi @pallakartheekreddy I have raised it for 5.4 and 5.6 , pls check and merge.

**Reply by pallakartheekreddy** [reply, marked as answer]
PR's merged. @santhosh-tg

**Comment by gohilamariappan** [comment]
@santhosh-tg Thank you!
